### ISSUE TITLE
Lack of BindExpression support in jsx-ast-utils breaking linter 

### ISSUE THREAD

**Issue by alexandr-g** [signavio/eslint-config-signavio#21]
that should be fixed in a newer version of `eslint-plugin-jsx-a11y` ( > 6 )

![screen shot 2017-10-11 at 10 25 21](https://user-images.githubusercontent.com/9251327/31429576-ba0a7240-ae6e-11e7-9109-46ba1831d1e8.png)

![screen shot 2017-10-11 at 10 30 02](https://user-images.githubusercontent.com/9251327/31429788-3f867770-ae6f-11e7-8995-84c9647eca8d.png)

![screen shot 2017-10-11 at 10 30 22](https://user-images.githubusercontent.com/9251327/31429806-47565e2a-ae6f-11e7-9f21-54ab5f798c62.png)
